x-pack/plugins/lens/public/shared_components/coloring/color_ranges/color_ranges_item_buttons.tsx

```diff
@@ -82,26 +82,28 @@ export function ColorRangeEditButton({
     });
   }, [isLast, dispatch, continuity, dataBounds, palettes]);
 
-  const title = i18n.translate('xpack.lens.dynamicColoring.customPalette.editButtonAriaLabel', {
-    defaultMessage: 'Edit',
-  });
+  let tooltipContent = isLast
+    ? i18n.translate('xpack.lens.dynamicColoring.customPalette.setCustomMinValue', {
+        defaultMessage: `Set custom maximum value`,
+      })
+    : i18n.translate('xpack.lens.dynamicColoring.customPalette.setCustomMaxValue', {
+        defaultMessage: `Set custom minimum value`,
+      });
+
+  if (disableSwitchingContinuity) {
+    tooltipContent = i18n.translate(
+      'xpack.lens.dynamicColoring.customPalette.disallowedEditMinMaxValues',
+      {
+        defaultMessage: `For current configuration you can not set custom value`,
```

Comment:
That's easier to read this way: 
```suggestion
        defaultMessage: `You cannot set custom value for current configuration`,
```

Reply:
I like it more, @flash1293 are you ok to update?

Reply:
Sounds good 👍